js/src/layers/ImageOverlay.js

```diff
@@ -51,4 +52,10 @@ export class LeafletImageOverlayView extends rasterlayer.LeafletRasterLayerView
       this
     );
   }
+
+  handle_message(content) {
+    if (content.msg == 'redraw') {
+      this.obj.redraw();
```

Comment:
There doesn't seem to be a `redraw` method on ImageOverlay according to the Leaflet docs https://leafletjs.com/reference.html#imageoverlay-setopacity
unless I missed it?

Reply:
Maybe you want to use `setUrl` instead, but there apparently no way of forcing the image to update if the url is the same.

According to this stackoverflow comment you can append the date to the url to force an update but I'm not sure we'd want to do this in ipyleaflet's codebase. Though nothing stops you from doing it in your notebook.

Reply:
Martin, thanks for the review.
For the leafletjs link to imageoverlay - I agree `redraw` is not a method on ImageOverlay, I will search through their github and see if that has been requested/makes sense as a feature request, if not I will file an issue.

The workarounds you suggested are ones I have employed and just seemed to add noise where noise was not needed - so I was trying to implement this redraw.

Since this would first depend on leaflet implementing something - should I just leave this pull request open as I work on that end?  Not sure what the proper protocol is for a pull request.

Reply:
> The workarounds you suggested are ones I have employed and just seemed to add noise where noise was not needed

Does the workaround work for the `ImageOverlay` then?

> Since this would first depend on leaflet implementing something - should I just leave this pull request open as I work on that end? 

Yes it's fine to keep this PR open for reference. I will turn it into draft.

Reply:
> Does the workaround work for the `ImageOverlay` then?
Yes, I was originally just taring down the layer and rebuilding as a workaround.  Your suggestion of just resetting the url is much cleaner - thank you.

It would be nice to not have to get into url management with a new image for each zoom set so I still filed the request on leaflet as that would simplify end-user code.
https://github.com/Leaflet/Leaflet/issues/8277

Reply:
One thing that would be nice would also to have the ImageOverlay work with an Image (from ipywidgets) widget